sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -325,7 +328,8 @@ def set_configuration_setting(
         :type configuration_setting: ~azure.appconfiguration.ConfigurationSetting
         :param match_condition: The match condition to use upon the etag
         :type match_condition: ~azure.core.MatchConditions
-        :keyword str etag: check if the ConfigurationSetting is changed. Set None to skip checking etag
+        :keyword str etag: check if the ConfigurationSetting is changed. \
+            Set None to use the value from keyword configuration_setting.
```

Comment:
I'm not sure we should tell users that they need to set None. Maybe we can just say "Uses the value from the configuration_setting if `etag` not set"

Reply:
Updated